### ISSUE TITLE
Looking for an ASUS FX553VE tester

### ISSUE THREAD

**Issue by arruor** [dominiksalvet/asus-fan-control#26]
As mentioned in #25 the FX553VE is a base for ROG Strix GL553VE and as they are sharing the same motherboards and instrumentation - they should have same temperature and fan sensors. As `asus-fan-control` runs out of the box on GL553VE it should be sufficient, for the upcoming tester,  to use following steps on FX553VE as well:

* Follow the [Instalaltion instructions](https://github.com/dominiksalvet/asus-fan-control#install) for `asus-fan-control`
* Follow the [steps provided](https://github.com/dominiksalvet/asus-fan-control/blob/master/contributing.md#basic-configuration) to obtain required information

**Comment by dominiksalvet** [comment]
Hello @arruor, I have modified your initial post significantly as we probably did not understand each other. Please, check it whether it is fine now. :smile:

For others, yes, **we are looking for the ASUS FX553VE tester**! Everything should go smoothly on your device. Please let us know! :+1: :rocket:

**Comment by arruor** [comment]
Hello @dominiksalvet,
now it makes more sense and I am happy with the changes.

**Comment by Bounty780** [comment]
> As `asus-fan-control` runs out of the box on GL553VE

For me, it doesn't. Could you help?

**Comment by arruor** [comment]
Hi @Bounty780 ,
Would you mind providing more information about your issue(s) please?
Thank you.

Here is my journey on fresh install on Ubuntu 21.04:

```
mind@hive:~$ sudo dmidecode -t 2
# dmidecode 3.3
Getting SMBIOS data from sysfs.
SMBIOS 3.0.0 present.

Handle 0x0002, DMI type 2, 15 bytes
Base Board Information
        Manufacturer: ASUSTeK COMPUTER INC.
        Product Name: GL553VE
        Version: 1.0
        Serial Number: <snip>
        Asset Tag: <snip>
        Features:
                Board is a hosting board
                Board is replaceable
        Location In Chassis: Default string
        Chassis Handle: 0x0003
        Type: Motherboard
        Contained Object Handles: 0

mind@hive:~$ sudo lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu Hirsute Hippo (development branch)
Release:        21.04
Codename:       hirsute
```

1. Install [GitPack](https://github.com/dominiksalvet/gitpack#installation)

```
mind@hive:~$ wget -qO- https://raw.githubusercontent.com/dominiksalvet/gitpack/master/.install/initl.sh | sh
installing GitPack locally
>>> running install for github.com/dominiksalvet/gitpack
downloading repository
reading status database
using automatic candidate version
installing version 1.0.0
running script .install/install-local
copying files
first local installation; computer reboot may be required
adding to status database
<<< done; successfully installed
success
```

2. Check installation path for GitPack

```
mind@hive:~$ which gitpack
/home/mind/.local/bin/gitpack
```

3. Install [asus-fan-control](https://github.com/dominiksalvet/asus-fan-control)

```
mind@hive:~$ sudo /home/mind/.local/bin/gitpack install github.com/dominiksalvet/asus-fan-control
>>> running install for github.com/dominiksalvet/asus-fan-control
downloading repository
reading status database
using automatic candidate version
installing version 3.10.0
running script .install/install-global-Linux
installing dmidecode package
installing acpi-call-dkms package
copying files
enabling systemd service
loading acpi_call module first time
applying fan speed changes
adding to status database
<<< done; successfully installed
```

4. Check `afc` service status

```
mind@hive:~$ sudo systemctl status afc
● afc.service - Executes asus-fan-control whenever necessary.
     Loaded: loaded (/etc/systemd/system/afc.service; enabled; vendor preset: enabled)
     Active: inactive (dead)
```

5. Enable `afc` service

```
mind@hive:~$ sudo systemctl enable afc
mind@hive:~$ sudo systemctl status afc
● afc.service - Executes asus-fan-control whenever necessary.
     Loaded: loaded (/etc/systemd/system/afc.service; enabled; vendor preset: enabled)
     Active: inactive (dead) since Sun 2021-03-14 14:10:06 GMT; 2s ago
    Process: 5883 ExecStartPre=/sbin/modprobe acpi_call (code=exited, status=0/SUCCESS)
    Process: 5884 ExecStart=/usr/local/bin/asus-fan-control (code=exited, status=0/SUCCESS)
   Main PID: 5884 (code=exited, status=0/SUCCESS)

Mar 14 14:10:06 hive systemd[1]: Starting Executes asus-fan-control whenever necessary....
Mar 14 14:10:06 hive systemd[1]: afc.service: Succeeded.
Mar 14 14:10:06 hive systemd[1]: Finished Executes asus-fan-control whenever necessary....
```

6. Set temperatures (please use values of your liking, I am using some sample values in my case):

```
mind@hive:~$ sudo asus-fan-control set-temps 45 50 55 60 65 70 75 80
mind@hive:~$ sudo asus-fan-control get-temps
45 50 55 60 65 70 75 80
```

7. Check fan speeds by using `sensors` command. If it's missing you have to install `lm-sensors` package

```
mind@hive:~$ sensors
coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +37.0°C  (high = +100.0°C, crit = +100.0°C)
Core 0:        +35.0°C  (high = +100.0°C, crit = +100.0°C)
Core 1:        +37.0°C  (high = +100.0°C, crit = +100.0°C)
Core 2:        +37.0°C  (high = +100.0°C, crit = +100.0°C)
Core 3:        +36.0°C  (high = +100.0°C, crit = +100.0°C)

pch_skylake-virtual-0
Adapter: Virtual device
temp1:        +43.0°C  

BAT0-acpi-0
Adapter: ACPI interface
in0:          16.37 V  
curr1:       170.00 mA 

iwlwifi_1-virtual-0
Adapter: Virtual device
temp1:        +35.0°C  

asus-isa-0000
Adapter: ISA adapter
cpu_fan:     2000 RPM

nvme-pci-0500
Adapter: PCI adapter
Composite:    +29.9°C  (low  = -273.1°C, high = +67.8°C)
                       (crit = +70.8°C)
Sensor 1:     +29.9°C  (low  = -273.1°C, high = +65261.8°C)
Sensor 2:     +31.9°C  (low  = -273.1°C, high = +65261.8°C)

acpitz-acpi-0
Adapter: ACPI interface
temp1:        +38.0°C  (crit = +98.0°C)
```

8. Put your laptop under some load - in my example I am running five tabs on Google Chrome browser playing same Full HD video

```
mind@hive:~$ ps afx|grep "chrome --type=renderer" | grep -v grep | wc -l
13

mind@hive:~$ sensors
coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +42.0°C  (high = +100.0°C, crit = +100.0°C)
Core 0:        +39.0°C  (high = +100.0°C, crit = +100.0°C)
Core 1:        +40.0°C  (high = +100.0°C, crit = +100.0°C)
Core 2:        +42.0°C  (high = +100.0°C, crit = +100.0°C)
Core 3:        +40.0°C  (high = +100.0°C, crit = +100.0°C)

pch_skylake-virtual-0
Adapter: Virtual device
temp1:        +44.0°C  

BAT0-acpi-0
Adapter: ACPI interface
in0:          16.36 V  
curr1:       288.00 mA 

iwlwifi_1-virtual-0
Adapter: Virtual device
temp1:        +39.0°C  

asus-isa-0000
Adapter: ISA adapter
cpu_fan:     2600 RPM

nvme-pci-0500
Adapter: PCI adapter
Composite:    +31.9°C  (low  = -273.1°C, high = +67.8°C)
                       (crit = +70.8°C)
Sensor 1:     +31.9°C  (low  = -273.1°C, high = +65261.8°C)
Sensor 2:     +33.9°C  (low  = -273.1°C, high = +65261.8°C)

acpitz-acpi-0
Adapter: ACPI interface
temp1:        +42.0°C  (crit = +98.0°C)
```

As you could see temperatures goes slightly up, hence the power consumption is increased from 170 mA to 288 mA accordingly. After I've stopped the Chrome browser I am getting values such as:

```

mind@hive:~$ ps afx|grep "chrome --type=renderer" | grep -v grep | wc -l
0

mind@hive:~$ sensors
coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +37.0°C  (high = +100.0°C, crit = +100.0°C)
Core 0:        +37.0°C  (high = +100.0°C, crit = +100.0°C)
Core 1:        +36.0°C  (high = +100.0°C, crit = +100.0°C)
Core 2:        +37.0°C  (high = +100.0°C, crit = +100.0°C)
Core 3:        +35.0°C  (high = +100.0°C, crit = +100.0°C)

pch_skylake-virtual-0
Adapter: Virtual device
temp1:        +43.0°C  

BAT0-acpi-0
Adapter: ACPI interface
in0:          16.29 V  
curr1:         0.00 A  

iwlwifi_1-virtual-0
Adapter: Virtual device
temp1:        +35.0°C  

asus-isa-0000
Adapter: ISA adapter
cpu_fan:     2000 RPM

nvme-pci-0500
Adapter: PCI adapter
Composite:    +29.9°C  (low  = -273.1°C, high = +67.8°C)
                       (crit = +70.8°C)
Sensor 1:     +29.9°C  (low  = -273.1°C, high = +65261.8°C)
Sensor 2:     +31.9°C  (low  = -273.1°C, high = +65261.8°C)

acpitz-acpi-0
Adapter: ACPI interface
temp1:        +38.0°C  (crit = +98.0°C)
```

Bottom line: You could try to experiment with different temperature ranges to find a setup which will suits you best.

Hope this would help.

**Comment by Bounty780** [comment]
You're ignoring the entire point here. I installed AFC, but it plainly and simple doesn't change fan speed.

**Comment by arruor** [comment]
I don't think I am ignoring anything.
I've just shared my experience installing AFC on a fresh Ubuntu install, as you've quoted my sentence.
As per example above - AFC worked out of the box for me at the time when I've created the issue, and also it is working for me at the moment [of writing], almost one year later.

I've asked for more details too - as we are not in your head and/or on your laptop, hence we couldn't know what the problems really is.

Would you mind sharing the results of following commands, please :

```
mind@hive:~$ lsmod |grep temp
x86_pkg_temp_thermal    20480  0
coretemp               20480  0

mind@hive:~$ sudo asus-fan-control get-temps
35 40 45 50 55 60 65 70

mind@hive:~$ sudo asus-fan-control model-info
name GL553VE
addresses 1335
default-temps 45 50 55 60 65 70 75 80
tested true

mind@hive:~$ sensors
coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +39.0°C  (high = +100.0°C, crit = +100.0°C)
Core 0:        +37.0°C  (high = +100.0°C, crit = +100.0°C)
Core 1:        +37.0°C  (high = +100.0°C, crit = +100.0°C)
Core 2:        +39.0°C  (high = +100.0°C, crit = +100.0°C)
Core 3:        +37.0°C  (high = +100.0°C, crit = +100.0°C)

pch_skylake-virtual-0
Adapter: Virtual device
temp1:        +44.5°C  

BAT0-acpi-0
Adapter: ACPI interface
in0:          16.26 V  
curr1:         0.00 A  

iwlwifi_1-virtual-0
Adapter: Virtual device
temp1:        +38.0°C  

asus-isa-0000
Adapter: ISA adapter
cpu_fan:     3200 RPM

nvme-pci-0500
Adapter: PCI adapter
Composite:    +32.9°C  (low  = -273.1°C, high = +67.8°C)
                       (crit = +70.8°C)
Sensor 1:     +32.9°C  (low  = -273.1°C, high = +65261.8°C)
Sensor 2:     +33.9°C  (low  = -273.1°C, high = +65261.8°C)

acpitz-acpi-0
Adapter: ACPI interface
temp1:        +39.0°C  (crit = +98.0°C)
```

Also you could find [afc-scout](https://github.com/dominiksalvet/afc-scout) helpful to get low-level information.
Given the results presented above `afc-scout` gave me:

```
mind@hive:~$ sudo afc-scout
------ base address 1335 ------
[1334] = 110
[1335] = 35 <- possible start
[1336] = 40
[1337] = 45
[1338] = 50
[1339] = 55
[1340] = 60
[1341] = 65
[1342] = 70 <- possible end
[1343] = 130
------ base address 1431 ------
[1430] = 12
[1431] = 45 <- possible start
[1432] = 84
[1433] = 84
[1434] = 84
[1435] = 84
[1436] = 84
[1437] = 84
[1438] = 84 <- possible end
[1439] = 130
```

Best regards.

**Comment by arruor** [comment]
Issue closed due to inactivity.

**Comment by dominiksalvet** [comment]
Honestly, even though the issue is inactive, I would let it open since AFC is still looking for the tester and that is the core of this issue... 😃

**Comment by dominiksalvet** [comment]
> Issue closed due to inactivity.

Okay, changed my mind to keep AFC issues clean and updated. :smile: Thank you for your assistance, maybe someone with FX553VE will get in touch with us later.

**Comment by Bounty780** [comment]
Since I came back to testing out Linux, I got these command outputs.

```
[root@strixik hwmon4]# lsmod |grep temp
x86_pkg_temp_thermal    20480  0
coretemp               20480  0
[root@strixik hwmon4]# sudo asus-fan-control get-temps
45 50 55 60 65 70 75 80
[root@strixik hwmon4]# sudo asus-fan-control model-info
name GL553VE
addresses 1335
default-temps 45 50 55 60 65 70 75 80
tested true
[root@strixik hwmon4]# sensors
iwlwifi_1-virtual-0
Adapter: Virtual device
temp1:        +51.0°C  

asus-isa-0000
Adapter: ISA adapter
cpu_fan:     4800 RPM

BAT0-acpi-0
Adapter: ACPI interface
in0:          16.14 V  
curr1:         0.00 A  

coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +72.0°C  (high = +100.0°C, crit = +100.0°C)
Core 0:        +69.0°C  (high = +100.0°C, crit = +100.0°C)
Core 1:        +71.0°C  (high = +100.0°C, crit = +100.0°C)
Core 2:        +68.0°C  (high = +100.0°C, crit = +100.0°C)
Core 3:        +69.0°C  (high = +100.0°C, crit = +100.0°C)

pch_skylake-virtual-0
Adapter: Virtual device
temp1:        +56.5°C  

acpitz-acpi-0
Adapter: ACPI interface
temp1:        +72.0°C  (crit = +98.0°C)

[root@strixik hwmon4]# sudo afc-scout
------ base address 1335 ------
[1334] = 110
[1335] = 45 <- possible start
[1336] = 50
[1337] = 55
[1338] = 60
[1339] = 65
[1340] = 70
[1341] = 75
[1342] = 80 <- possible end
[1343] = 130
------ base address 1431 ------
[1430] = 0
[1431] = 45 <- possible start
[1432] = 84
[1433] = 84
[1434] = 84
[1435] = 84
[1436] = 84
[1437] = 84
[1438] = 84 <- possible end
[1439] = 130
```

**Comment by Bounty780** [comment]
So, I suppose AFC does "something" now?

Now when I booted my PC, the fan speed was a lot better than before, and sometimes it spikes to the speed I last had on Windows, but then it quickly clocks back.

**Comment by dominiksalvet** [comment]
@Bounty780 Please create a new issue, so we do not clutter the current one with irrelevant information...
